webapp/src/styles/_global.scss

```diff
@@ -1,25 +1,27 @@
-.plugin-btn {
-    &:focus {
-        outline: none !important;
+.azure-devops-plugin {
+    .plugin-btn {
+        &:focus {
+            outline: none !important;
+        }
     }
-}
-
-.project-details {
-    flex-basis: 70%;
-    overflow: hidden;
-}
-
-.button-wrapper {
-    flex-basis: 30%;
-}
-
-.rhs-wrapper {
-    display: flex;
-    flex-direction: column;
-    width: 100%;
-    height: calc(100vh - 120px); // 120px is the height of button wrapper
-
-    &__content {
-        overflow: auto;
+    
+    .project-details {
+        flex-basis: 70%;
+        overflow: hidden;
+    }
+    
+    .button-wrapper {
+        flex-basis: 30%;
     }
+    
+    .rhs-wrapper {
+        display: flex;
+        flex-direction: column;
```

Comment:
We can use util classes for this.

Reply:
this is a global style and here we have just overridden some stylings of different classes meaning if any class of this file is used anywhere within the code then we are targeting them and adding some common/global styles at just one place

putting utils classes in such cases will tend to add that util class at all those places wherever needed